core/src/navigation/GlobalSearch.html

```diff
@@ -127,18 +127,18 @@
   });
 
   function getSearchPlaceholder(search) {
```

Comment:
```suggestion
  function getSearchPlaceholder({ searchProvider }) {
```
There are a lot of repeats of `search.searchProvider`
Either destructuring here or pass `searchProvider` directly e.g. `getSearchPlaceholder(search.searchProvider)`
at https://github.com/SAP/luigi/blob/868a21c6f9cdb4c4b9b269ea74ecdb9022ddf715/core/src/navigation/GlobalSearch.html#L117